client/hotrod-client/src/main/java/org/infinispan/client/hotrod/RemoteCacheManagerAdmin.java

```diff
@@ -25,6 +27,18 @@
    @Override
    <K, V> RemoteCache<K, V> createCache(String name, String template) throws HotRodClientException;
 
+   /**
+    * Creates a cache on the remote server cluster using the specified configuration
+    *
+    * @param name the name of the cache to create
+    * @param configuration a concrete configuration that will be sent to the server. The XML representation of this
+    *                      configuration must be in a format supported by the server.
```

Comment:
It's not clear what is the XSD of the configuration that must be passed in. Is it ```urn:infinispan:config:9.2``` (embedded)  or  ```urn:infinispan:server:core:9.2``` (server)? To avoid any doubt, I think we should mention in the javadoc

Reply:
I'd really like to have a common schema which is then xincluded into the two subschemas (embedded and server) which can then add their own idiosyncrasies. But at the moment this is the embedded schema.

Reply:
that's fine, IMO we should just mention that in the javadoc